_data/projects/AMPProject.yml

```diff
@@ -0,0 +1,13 @@
+name: The AMP Project (Accelerated Mobile Pages)
+desc: The AMP Project provides a web component library to make it easy to build fast web pages.
+site: https://github.com/ampproject/amphtml/
```

Comment:
Suggestion: if the home page for the project is more interesting as an entry point, you can use that here. I'm going to merge this anyway, but feel free to submit another PR to tweak that based on how it looks on the site.